tests/SendSalesCommandTest.php

```diff
@@ -107,18 +107,25 @@
         config([
             'trx_mall_upload_sales_data_api.notifications.mail.name' => $this->mailConfig['name'],
             'trx_mall_upload_sales_data_api.notifications.mail.email' => $this->mailConfig['email'],
+            'trx_mall_upload_sales_data_api.notifications.mail.enable_failure_notifications_only' => true,
         ]);
     });
 
-    it('sends failure notification when no stores found', function () {
+    it('sends failure notification when no stores found', function ($isEnableFailureNotificationsOnly) {
+
+        config()->set('trx_mall_upload_sales_data_api.notifications.mail.enable_failure_notifications_only', $isEnableFailureNotificationsOnly);
 
         $this->serviceMock->shouldReceive('getStores')->once()
             ->andReturn([]);
 
         artisan('trx:send-sales', [
             '--date' => '2024-01-11',
         ])->assertExitCode(1);
```

Comment:
Please check whether the notification is triggered/not triggered based on the data set provided.

Reply:
I am curious. Why that code was there already? Was it being used?

Reply:
Yes. it was there already. That particular test is being used for testing notification.